mod-vendors/schemas/vendor.json

```diff
@@ -167,7 +167,167 @@
+          "type": "string"
+        },
+        "prorate_tax": {
+          "description": "The setting to prorate tax for this EDI",
+          "type": "boolean"
+        },
+        "prorate_fees": {
+          "description": "The setting to prorate fees for this EDI",
+          "type": "boolean"
+        },
+        "edi_naming_convention": {
+          "description": "The naming convention for this EDI",
+          "type": "string"
+        },
+        "send_acct_num": {
+          "description": "The setting to send the account number for this EDI",
+          "type": "boolean"
+        },
+        "support_order": {
+          "description": "The setting to support order for this EDI",
+          "type": "boolean"
+        },
+        "support_invoice": {
+          "description": "The setting to support invoice for this EDI",
+          "type": "boolean"
+        },
+        "notes": {
+          "description": "The notes for this EDI",
+          "type": "string"
+        },
+        "edi_ftp": {
+          "description": "The FTP object for this EDI",
+          "type": "object",
+          "properties": {
+            "ftp_format": {
+              "description": "The FTP format for this EDI",
+              "type": "string"
+            },
+            "server_address": {
+              "description": "The server address for this EDI",
+              "type": ["string", "null"],
+              "pattern": "^([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$"
+            },
+            "username": {
+              "description": "The login username for this EDI",
+              "type": "string"
+            },
+            "password": {
+              "description": "The login password for this EDI",
+              "type": "string"
+            },
+            "ftp_mode": {
+              "description": "The FTP mode for this EDI",
+              "type": "string"
+            },
+            "ftp_conn_mode": {
+              "description": "The FTP connection mode for this EDI",
+              "type": "string"
+            },
+            "ftp_port": {
+              "description": "The port for this EDI",
+              "type": "string"
+            },
+            "order_directory": {
+              "description": "The order directory for this EDI",
+              "type": "string"
+            },
+            "invoice_directory": {
+              "description": "The invoice directory for this EDI",
+              "type": "string"
+            },
+            "notes": {
+              "description": "The notes for this EDI",
+              "type": "string"
+            }
+          }
+        },
+        "edi_job": {
+          "description": "The job object for this EDI",
+          "type": "object",
+          "properties": {
+            "schedule_edi": {
+              "description": "The schedule for this EDI job",
+              "type": "boolean"
+            },
+            "date": {
```

Comment:
I'm guessing this corresponds to the "scheduling_date" field in https://docs.google.com/spreadsheets/d/1F88PoSnEKD7oFnza1ZuOgzAgKnK95XOyTQzxyww3tZA/edit?ts=5c1bd54f#gid=0 

We might want to get some clarification from Dennis on A) the format of this field and B) a better description to use in the schema.  The way it stands now with "date" and "time" fields both of which are "date-time" it's unclear what the intent is.

Reply:
This is the intended start date of the EDI job.  Format was taken from the UI form and included in the property description.  Name updated to 'scheduling_date'.